html-aam/customized-select/optgroup-name-desc.tentative.html

```diff
@@ -0,0 +1,265 @@
+<!doctype html>
+<html lang=en>
+<head>
+  <title>Tentative: HTML-AAM Optgroup Within Custom Styled Select Naming Verification Tests</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/resources/testdriver.js"></script>
+  <script src="/resources/testdriver-vendor.js"></script>
+  <script src="/resources/testdriver-actions.js"></script>
+  <script src="/wai-aria/scripts/aria-utils.js"></script>
+  <style>
+    /* enable the customized select */
+    select, ::picker(select) {
+      appearance: base-select;
+    }
+  </style>
+</head>
+<body>
+
+  <p>Tests the accName for the <code>optgroup</code> element, within a customizable <code>select</code> element as defined in <a href="https://w3c.github.io/html-aam/#accessible-name-and-description-computation">HTML-AAM: Accessible Name Computations By HTML Element</a>.
+  These tests are meant to show whether an element returns a name per the naming mechanism used. See <a href="https://wpt.fyi/results/accname">wpt: accname</a> for expanded accName testing.</p>
+
+  <!--
+    Tests the expected accessible name for an optgroup 
+    see: https://github.com/w3c/aria/pull/2360
+  -->
+
+<!-- aria-labelledby tests:
+  
+test 1: name = foo
+        description = n/a
+-->
+<select>
+    <button><selectedcontent></selectedcontent></button>
+    <optgroup aria-labelledby=f data-testname="optgroup aria-labelledby" data-expectedlabel="foo" class="ex">
+       <div id=f>foo</div> <!-- invalid per content model -->
```

Comment:
This div is just invalid because it has text in it, right?

Reply:
yes

Reply:
added a comment on 254 to call out an instance of an empty div as being valid under the content model